codespell_lib/data/dictionary.txt

```diff
@@ -19896,7 +20094,9 @@ wonderfull->wonderful
 wont't->won't
 wont->won't, wont,
 worarounds->workarounds
+wordcamps->wordcamp
```

Comment:
`wordcamps` is a plural of `wordcamp` right?

Reply:
Removed